app/controllers/users_controller.rb

```diff
@@ -2,22 +2,16 @@ class UsersController < ApplicationController
   before_filter :authenticate_user!
 
   def index
-    @search = User.ransack(params[:q])
-    @search.sorts = 'members_member_uid asc' if @search.sorts.empty?
-
-    @users = @search
-      .result(distinct: false)
-      .joins(members: :account)
-      .eager_load(members: :account)
-      .where(members: { organization: current_organization.id })
-      .page(params[:page])
-      .per(25)
-
-    @memberships = current_organization.members.
-      where(user_id: @users.map(&:id)).
-      includes(:account).each_with_object({}) do |mem, ob|
-        ob[mem.user_id] = mem
-      end
+    default_sort
+
+    @search =
+      current_organization.members.joins(:account, :user).ransack(params[:q])
+
+    @members =
+      @search.result.page(params[:page]).per(25)
+
+    @member_view_models =
```

Comment:
There is no easy way to keep the pagination info exposed and at the same time to decorate the objects. It's this or another "paginated collection decorator", which most of the times I find ugly.